tests/int/tests/bootstrap.js

```diff
@@ -16,7 +16,7 @@ describe('Bootstrapping', () => {
       .then(() => hortiUtils.start([
         '--install=test:test-app-1:1.0.0',
         '--no-daemon',
-        '--test'], { waitUntil: true }))
+        '--test'], { waitUntil: true, log: true }))
```

Comment:
NB: if you didn't mean to leave there here... you did!

Reply:
I meant to leave that there!